tests/testthat/test-submit-models.R

```diff
@@ -53,14 +59,45 @@ withr::with_options(list(rbabylon.bbi_exe_path = "bbi",
               # check each call
               expect_identical(
                 proc_list[[1]][[PROC_CALL]],
-                as.character(glue("cd {file.path(getwd(), MODEL_DIR)} ; bbi nonmem run sge {CTL_FILENAME} 2.ctl --overwrite --threads=1"))
+                as.character(
+                  glue(
+                    "cd {model_dir} ; bbi nonmem run sge {mod_ctl_path[1]} {mod_ctl_path[2]} --overwrite --threads=1"
+                  )
+                )
               )
               expect_identical(
                 proc_list[[2]][[PROC_CALL]],
-                as.character(glue("cd {file.path(getwd(), MODEL_DIR)} ; bbi nonmem run sge 3.ctl --clean_lvl=2 --overwrite --threads=1"))
+                as.character(
+                  glue("cd {model_dir} ; bbi nonmem run sge {mod_ctl_path[3]} --clean_lvl=2 --overwrite --threads=1"))
               )
             })
 
+  test_that("submit_models() works for models in different directories", {
+    new_dir <- "level2"
```

Comment:
Let's add a TODO here (or somewhere in this test) to the effect of "once #174 is done we should revisit using `create_all_models()` from `setup-workflow-ref.R` instead of manually calling `copy_model_from()` all over this test."